cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -456,9 +456,13 @@ public static KafkaCluster fromCrd(Kafka kafkaAssembly, KafkaVersion.Lookup vers
         result.setListeners(listeners);
 
         if (listeners != null) {
-            if (listeners.getPlain() != null
-                    && listeners.getPlain().getAuth() instanceof KafkaListenerAuthenticationTls) {
-                throw new InvalidResourceException("You cannot configure TLS authentication on a plain listener.");
+            if (listeners.getPlain() != null) {
```

Comment:
OAuth works on `tls` and `external` as well. We need to validate it there as well.

Reply:
This still needs the TLS listener validation, or?